app/assets/stylesheets/views/_marketplace_landing_page.scss

```diff
@@ -306,12 +306,24 @@ $link-color: rgb(3,102,214);
     position: relative;
     flex: 1 1 100%;
     margin: 1em 0 2em 0;
+    @include source-neue-kabel;
+    font-size: 18px;
+    line-height: 23px;
 
     @include media(tablet) {
       max-width: 45.5%;
     }
   }
 
+  .how-it-works-details span {
```

Comment:
you should use a class selector here, instead of the ```span``` tag. It's more readable, efficient, and flexible if you change things. https://css-tricks.com/efficiently-rendering-css/